controllers/reconcilers/mm_inferenceservice_reconciler.go

```diff
@@ -17,52 +17,45 @@ package reconcilers
 
 import (
 	"context"
+	"github.com/hashicorp/go-multierror"
 
 	"github.com/go-logr/logr"
 	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
 	"github.com/opendatahub-io/odh-model-controller/controllers/utils"
-	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var _ Reconciler = (*ModelMeshInferenceServiceReconciler)(nil)
+
 type ModelMeshInferenceServiceReconciler struct {
-	client                       client.Client
-	routeReconciler              *ModelMeshRouteReconciler
-	serviceAccountReconciler     *ModelMeshServiceAccountReconciler
-	clusterRoleBindingReconciler *ModelMeshClusterRoleBindingReconciler
+	SingleResourcePerNamespace
```

Comment:
`ModelMeshInferenceServiceReconciler` should not implement `SingleResourcePerNamespace` interface because this class doesn't created any resource directly. It just a wrapper to call `ModelMeshRouteReconciler`, `ModelMeshServiceAccountReconciler` & `ModelMeshClusterRoleBindingReconciler`

Reply:
Correct, it's not subresource reconciler. I think it's leftover. Good catch! Cleaned in 4a9f022